### PR TITLE
Feature/initial velocity directives

### PR DESCRIPTION
**Pull request by jamesjwarren**
Initial PR with velocity directives. 

Do we need all the angular-seed info in the README, I don't think all of it is relevant (esp the app structure)?

Still need to add tests.

### CODE REVIEW

File: app/js/directives/snVelocity.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"use strict";
```
Comment (edoparearyee) [inline app/js/directives/snVelocity.js]: I don't think it's necessary to use namespace for filenames you can just call the files `velocity.js` and `velocity-group.js`

http://stackoverflow.com/questions/7273316/what-is-the-javascript-filename-naming-convention

File: app/js/sn.velocity.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"use strict";
```
Comment (edoparearyee) [inline app/js/sn.velocity.js]: same here you could just use `app.js` or `angular-velocity.js`

**Comment by edoparearyee** [comment]
@jamesjwarren I think the angular seed stuff is useful for someone not familiar to node, grunt, bower etc... who wants to add stuff to this lib, if theres anything that isn't accurate like the folder structure then you should update it. The README should include instructions on how to use the library e.g. through bower and how to build the app, run tests etc..

**Comment by jamesjwarren** [comment]
Updated and added unit tests, so if it passes should ok to merge.

**Comment by edoparearyee** [comment]
@jamesjwarren looks good, you should do the PR into the develop branch

**Pull request closed without merging**